styles/japanesewithanime.com/catppuccin.user.css

```diff
@@ -78,7 +78,7 @@
 	body *::-webkit-scrollbar-button{
 		color: @text !important;
 		background-color: @base !important;
-		border: 0px  !important;
+		border-width: 0 !important;
```

Comment:
This changes the layout of the page in most cases. How does the website theme it originally?

Reply:
im not even sure what that does and removing it doesn't seem to break anything nor does change any values

Reply:
...why did you put it there then lol?

Reply:
i can't remember, i think i was having trouble theming the sidebar separator between the main page and the sidebar maybe it got added there?